_layouts/indicator.html

```diff
@@ -104,9 +105,19 @@ <h4>{{ t.indicator.sub_categories }}</h4>
                 <img src="{{ site.baseurl }}/assets/img/loading.gif" alt="{{ t.indicator.loading_map }}" />
               </div>
             </div>
-            {% if meta.embedded_map_html %}
+            {% if meta.embedded_feature_url %}
             <div role="tabpanel" class="tab-pane" id="embeddedmapview" class="embedded-map">
-              {{ meta.embedded_map_html }}
+              <div role="tabpanel" class="tab-pane" id="embeddedmapview" >
+              <div class="row">
+              <div class="col-xs-12">
+                <h3 tabindex="0">{{ meta.embedded_feature_title }}</h3>
```

Comment:
Running this through the `t` filter will allow this to support the possibility of translation: `{{ meta.embedded_feature_title | t }}`